modules/data-model/links/impl/src/main/java/io/uhndata/cards/links/internal/serialize/LinksProcessor.java

```diff
@@ -51,6 +51,12 @@ public String getName()
         return "links";
     }
 
+    @Override
+    public String getDescription()
+    {
+        return "Simplify link serialization to only include the link type and path to the linked resource";
```

Comment:
Do we support links to anything or just forms?
```suggestion
        return "Simplify the serialization of links to other resources (such as other forms) to only include the link type and path to the linked resource.";
```